### PR TITLE
Migrate Pricing Comparison to Docs

### LINKED ISSUE #1823: Platform Resources by Plan Doc Update
Re: [Platform Resources by Plan](https://pantheon.io/docs/platform-resources)

Refactor doc and preserve all content from [Pricing Comparison](https://pantheon.io/pricing-comparison)

### PR DESCRIPTION
**Pull request by rachelwhitton**
Closes #1823 
## Effect

PR includes the following changes:
- Refactor existing doc 
- Preserve table from [Pricing Comparison](https://pantheon.io/pricing-comparison) page
- CSS tweaks 
## Remaining Work
- [x] Audit existing content as suggested 

cc @joshkoenig 
@nataliejeremy

### CODE REVIEW

File: source/_docs/platform-resources.md [hunk 1]
```diff
@@ -1,75 +1,275 @@
...
+      <td>Not available</td>
+      <td>Not available</td>
+      <td>Not available</td>
+      <td>Available</td>
+    </tr>
+    <tr>
+      <td class="features">Support Tickets <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-content="Access to website experts to help with launch and ongoing site performance."><em class="fa fa-info-circle" style="color:#aab3b9;" ></em></a></td>
+      <td>Included</td>
+      <td>Included</td>
+      <td>Unlimited tickets Priority treatment</td>
```
Comment (nataliejeremy) [inline source/_docs/platform-resources.md]: Recommend using "Unlimited tickets" and removing the "priority treatment" part

File: source/_docs/platform-resources.md [hunk 2]
```diff
@@ -1,75 +1,275 @@
...
+    </tr>
+    <tr>
+      <td class="features">Phone Support <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-content="One-on-one conversation to guide you through unresolved tickets or complicated website issues."><em class="fa fa-info-circle" style="color:#aab3b9;" ></em></a></td>
+      <td>Not available</td>
+      <td>Not available</td>
+      <td>As required</td>
+      <td>Included</td>
+    </tr>
+    <tr>
+      <td class="features">24x7 On-call <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-content="The Elite customer bat-phone—emergency support anytime, anywhere for your website."><em class="fa fa-info-circle" style="color:#aab3b9;" ></em></a></td>
```
Comment (nataliejeremy) [inline source/_docs/platform-resources.md]: This is now called Pantheon On-Call

File: source/_docs/platform-resources.md [hunk 3]
```diff
@@ -1,75 +1,275 @@
...
+      <td>Not available</td>
+      <td>Not available</td>
+      <td>Available</td>
+    </tr>
+    <tr>
+      <td class="features">Support Tickets <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-content="Access to website experts to help with launch and ongoing site performance."><em class="fa fa-info-circle" style="color:#aab3b9;" ></em></a></td>
+      <td>Included</td>
+      <td>Included</td>
+      <td>Unlimited tickets Priority treatment</td>
+      <td>Unlimited tickets 2 hour response SLA-guaranteed</td>
```
Comment (nataliejeremy) [inline source/_docs/platform-resources.md]: Suggest: Unlimited tickets - 2 hour response

File: source/_docs/platform-resources.md [hunk 4]
```diff
@@ -1,75 +1,275 @@
...
+<table class="table  table-bordered table-responsive table-striped plan-comparison">
+    <thead>
       <tr>
-        <th scope="row" class="thead-inverse"></th>
-        <th>Personal</th>
-        <th>Professional</th>
-        <th>Business</th>
-        <th>Elite</th>
+        <th class="subscription"></th>
+        <th class="subscription">Personal</th>
```
Comment (nataliejeremy) [inline source/_docs/platform-resources.md]: This column seems a bit squished and some of the content wraps to the second line

**Comment by joshkoenig** [comment]
We'll need to hold on this for the pricing page redesign on the main website.

Also, I think we should consider freeing ourselves from the particular design to make this more maintainable and "tech reviewer" friendly.

**Comment by nataliejeremy** [comment]
@ari-gold We need to make sure this info is up to date. Can you point me to the right person if it isn't you?

File: source/_docs/platform-resources.md [hunk 5]
```diff
@@ -1,75 +1,275 @@
...
+    </tr>
+    <tr>
+      <td class="features">Storage <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-content="Robust SSD storage across the entire platform."><em class="fa fa-info-circle" style="color:#aab3b9;" ></em></a></td>
+      <td>Up to 5GB</td>
+      <td>20GB</td>
+      <td>30GB</td>
+      <td>100GB+</td>
+    </tr>
+    <tr>
+      <td class="features">Application Memory Limit <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-content="The amount of memory any single request can utilize. Sites that handle complex data, or with very large codebases will need more."><em class="fa fa-info-circle" style="color:#aab3b9;" ></em></a></td>
```
Comment (ari-gold) [inline source/_docs/platform-resources.md]: Application Memory Limit >> PHP Memory Limit

File: source/_docs/platform-resources.md [hunk 6]
```diff
@@ -1,75 +1,275 @@
...
+    </tr>
+    <tr>
+      <td class="features">Edge Cache <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-content="Our edge-cache uses Varnish to accelerates content and pages on every Pantheon-powered site."><em class="fa fa-info-circle" style="color:#aab3b9;" ></em></a></td>
+      <td>Included</td>
+      <td>Included</td>
+      <td>Included</td>
+      <td>Included</td>
+    </tr>
+    <tr>
+      <td class="features">Application Cache <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-content="Our application cache uses Redis to significantly accelerate sites that make heavy use of the database."><em class="fa fa-info-circle" style="color:#aab3b9;" ></em></a></td>
```
Comment (ari-gold) [inline source/_docs/platform-resources.md]: Application Cache >> Redis Cache

File: source/_docs/platform-resources.md [hunk 7]
```diff
@@ -1,75 +1,275 @@
...
+    </tr>
+    <tr>
+      <td class="features">Site Monitoring <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-content="We monitor your site every minute from around the world and can take action if downtime is detected."><em class="fa fa-info-circle" style="color:#aab3b9;" ></em></a></td>
+      <td>Not available</td>
+      <td>Included</td>
+      <td>Included</td>
+      <td>Included with on-call alerting</td>
+    </tr>
+    <tr>
+      <td class="features">Full-text Search <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-content="Pantheon provides a full Solr stack as a service, enabling performant, faceted full-text search of content and documents."><em class="fa fa-info-circle" style="color:#aab3b9;" ></em></a></td>
```
Comment (ari-gold) [inline source/_docs/platform-resources.md]: Full-text Search >> Apache Solr Search Indexing

File: source/_docs/platform-resources.md [hunk 8]
```diff
@@ -1,75 +1,275 @@
 ---
-title: Platform Resources by Plan
+title: Platform Resources and Features by Plan
```
Comment (ari-gold) [inline source/_docs/platform-resources.md]: Site Resources and Features by Plan

File: source/_docs/platform-resources.md [hunk 9]
```diff
@@ -1,75 +1,275 @@
...
+    </tr>
+    <tr>
+      <td class="features">Cloud Development Environments <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-content="The Multidev feature lets you spin up Git feature branches as cloud development environments for teams."><em class="fa fa-info-circle" style="color:#aab3b9;" ></em></a></td>
+      <td>Not available</td>
+      <td>Not available</td>
+      <td>Up to 5</td>
+      <td>Up to 10</td>
+    </tr>
+    <tr>
+      <td class="features">Secure Runtime Access <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-content="Force all developer access to site resources through SSH tunnel."><em class="fa fa-info-circle" style="color:#aab3b9;" ></em></a></td>
```
Comment (ari-gold) [inline source/_docs/platform-resources.md]: Remove this row

File: source/_docs/platform-resources.md [hunk 10]
```diff
@@ -1,75 +1,275 @@
...
+    </tr>
+    <tr>
+      <td class="features">Roles and Permissions <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-content="Feature Change Management allows you to restrict access to deploying in test and live environments to only trusted users."><em class="fa fa-info-circle" style="color:#aab3b9;" ></em></a></td>
+      <td>Not available</td>
+      <td>Not available</td>
+      <td>Not available</td>
+      <td>Included</td>
+    </tr>
+    <tr>
+      <td class="features">Secure Integration <a rel="popover" data-proofer-ignore data-toggle="tooltip" data-html="true" data-content="Pantheon Enterprise Gateway allows you to securely connect to resources that reside behind a firewall."><em class="fa fa-info-circle" style="color:#aab3b9;" ></em></a></td>
```
Comment (ari-gold) [inline source/_docs/platform-resources.md]: Secure Integration >> Pantheon Enterprise Gateway

**Comment by ari-gold** [comment]
We'd like this to serve as a tool to see how plans are different, so let's remove all the rows were everything is "Included"

**Comment by nataliejeremy** [comment]
Thanks @ari-gold I updated it with your feedback. @rachelwhitton any ideas on the layout? Content  is updated.

**Pull request closed without merging**